docs/develop/bind-to-regions-in-a-document-or-spreadsheet.md

```diff
@@ -86,21 +103,19 @@ function write(message){
 }
 ```
 
-In this example, the specified binding type is text. This means that a [TextBinding] will be created for the selection that the user specifies in the prompt.
+The binding type is text, so a [TextBinding] will be created for the user's selection in the prompt.
 
-The second parameter is an object that contains the ID of the new binding being created. If an ID is not specified, one is generated automatically.
+The second parameter contains the ID of the new binding. If you don't specify an ID, one is generated automatically.
 
-The anonymous function passed into the method as the third _callback_ parameter is executed when the creation of the binding is complete. When the callback function executes, the [AsyncResult] object contains the status of the call and the newly created binding.
+The anonymous function passed as the third _callback_ parameter runs when the binding creation is complete. When the callback function runs, the [AsyncResult] object contains the call's status and the newly created binding.
 
-Figure 1 shows the built-in range selection prompt in Excel.
-
-*Figure 1. Excel Select Data UI*
+The following figure shows the built-in range selection prompt in Excel.
 
 ![The Select Data dialog.](../images/agave-api-overview-excel-selection-ui.png)
```

Comment:
I noticed lots of images are named agave- something. This is minor but it is a wasted SEO opportunity. If possible, I recommend renaming the files to office-add-in something.